### PR TITLE
Updating bitcoin_ko_KR.ts

### PR DESCRIPTION
**Pull request by steve-cw-chung**
added all the missing translations and fixed incorrect translations

### CODE REVIEW

**Comment by patricklodder** [comment]
I'll look into the conflicts and help out with those over the next couple of days

**Comment by dogespacewizard** [comment]
glad that you turned up @steve-cw-chung thank you for the korean translations!

File: src/qt/locale/bitcoin_ko_KR.ts [hunk 1]
```diff
@@ -1403,226 +1403,230 @@
...
 p, li { white-space: pre-wrap; }
 &lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:&apos;Sans Serif&apos;; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
 &lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
+        &lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
+        p, li { white-space: pre-wrap; }
+        &lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:&apos;Sans Serif&apos;; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
+        &lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</translation>
```
Comment (huedaya) [inline src/qt/locale/bitcoin_ko_KR.ts:1529]: I think this line is unnecessary since you're not translating any words?

Comment (steve-cw-chung) [reply src/qt/locale/bitcoin_ko_KR.ts:1529]: I believe it is necessary to be consistent with the style on Korean translated page. Unless you want the Korean page to have a different style.

Comment (huedaya) [reply src/qt/locale/bitcoin_ko_KR.ts:1529]: Alight :+1:

**Comment by rnicoll** [comment]
I had a bit of a think about how to resolve this - problem is, we have two translations in places and I can't judge which is better. I've made https://github.com/dogecoin/dogecoin/pull/2822 just so we've got the diff visible, but the suggestion there is make a PR which merges _only_ the non-conflicting parts, and then huedaya raise any intentional changes to the translation as a follow-on PR.

WDYT Patrick?

**Comment by patricklodder** [comment]
I was thinking to just push a rebase of this to `steve-cw-chung:master`.

**Comment by steve-cw-chung** [comment]
Let me know if you need me to update anything.

**Review by Bartascnk — APPROVED** [review]

**Review by Bartascnk — APPROVED** [review]

File: src/qt/locale/bitcoin_ko_KR.ts [hunk 2]
```diff
@@ -17,11 +17,11 @@
     </message>
     <message>
         <source>Enter the peer&apos;s address</source>
-        <translation type="unfinished"></translation>
+        <translation>피어의 주소를 입력하세요</translation>
     </message>
     <message>
         <source>Enter the peer&apos;s port</source>
-        <translation type="unfinished"></translation>
```
Comment (Bartascnk) [inline src/qt/locale/bitcoin_ko_KR.ts:24]: 1

Comment (Bartascnk) [inline src/qt/locale/bitcoin_ko_KR.ts:24]: Bartascnk@gmail.com

**Review by Bartascnk — APPROVED** [review]

**Comment by Bartascnk** [comment]
0xa4e8c3ec456107ea67d3075bf9e3df3a75823db00xa4e8c3ec456107ea67d3075bf9e3df3a75823db0

**Comment by QuentinFlores** [comment]
I am not happy with Paxful. After depositing some funds to my paxful account, they prompted me to send my ID for verification. I did and they rejected my ID for verification of my account and also Froze My Paxful account. I had a huge some of amount in that account and it was almost a month with no response from them until a colleague recommended JimFundsRecovery at Consultant dot Com, within few days they recovered all funds and logins to my paxful account, really appreciate

**Comment by QuickMythril** [comment]
@QuentinFlores - no scam posting.  you have been reported.

**Review by CryptoUnchained — APPROVED** [review]

**Review by CryptoUnchained — APPROVED** [review]

File: src/qt/locale/bitcoin_ko_KR.ts (hunk 1 above)
Comment (CryptoUnchained) [reply src/qt/locale/bitcoin_ko_KR.ts:1529]: doge

**Review by Chutancdn — APPROVED** [review]

**Review by patricklodder — APPROVED** [review]
Since this has seen some native speaker review, ACK from me.

**Pull request merged**